### PR TITLE
Devlinereturn

### PR DESCRIPTION
**Pull request by XanderSuFromQBIX**
Made it pretty

### CODE REVIEW

File: Moonrake/Program.cs [hunk 1]
```diff
@@ -10,6 +10,7 @@ class Program
     {
         static void Main(string[] args)
         {
+
```
Comment (Materia-xx) [inline Moonrake/Program.cs:13]: Do we really need an extra line return here?

**Review by Materia-xx — APPROVED** [review]

File: Moonrake/Program.cs (hunk 1 above)
Comment (XanderSuFromQBIX) [reply Moonrake/Program.cs:13]: Yes, WE DO!!

Comment (StickWolf) [reply Moonrake/Program.cs:13]: it workssss

**Review by StickWolf — APPROVED** [review]
hahaha

**Pull request merged**